docs/algorithms/DifferentialConflation.asciidoc

```diff
@@ -9,6 +9,6 @@ take two datasets, A and B, and generate a changeset from them such that the cha
 includes all of the data in set B that does not overlap (conflict, need merging, etc.) with
 set A. The changeset can then be applied directly to dataset A.
 
-This algorithm is still under development. Currently it works best if you use the Network Conflation conflicts matcher (see conf/core/Network.conf). In a nutshell, both maps are loaded, matches are generated, then all matches from map1 are removed - leaving you with all of map2, plus the "extra stuff" from map1.
+This algorithm is still under development. Currently it works best if you use the Network Conflation conflicts matcher (see conf/core/Network.conf). In a nutshell, both maps are loaded, matches are generated, then all of the elements involved in the matches are removed from the map. Then, in a secondary step, all of the remaing elements from the first map are removed.
```

Comment:
`remaining` because Dean once docked me for misspelling theta in source code comments...

Reply:
@mschicker I would just make sure that the PR pipeline succeeds and then fix the spelling error and commit.  Don't worry about having to redo all of the checks.

Reply:
Tough but fair. Thanks!